src/ticket/lib/ticketReducer.ts

```
@@ -0,0 +1,78 @@
+          ? { ...ticket, status: ticket.status === 'open' ? 'closed' : 'open' }
+          : ticket,
+      );
+      return newTickets;
+    case 'ADD_COMMENT':
+      const commentIds = tickets.reduce(
+        (commentIds, ticket) => [
+          ...commentIds,
+          ...ticket.comments.map((comment) => comment.id),
+        ],
+        [] as number[],
+      );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)